boot/zephyr/flash_map_extended.c

```diff
@@ -25,7 +25,7 @@ BOOT_LOG_MODULE_DECLARE(mcuboot);
 #define FLASH_DEVICE_ID SPI_FLASH_0_ID
 #if DT_NODE_HAS_STATUS(DT_INST(0, st_stm32_xspi_nor), okay)
 #define FLASH_DEVICE_NODE DT_INST(0, st_stm32_xspi_nor)
-#define FLASH_DEVICE_BASE DT_REG_ADDR(DT_INST(0, st_stm32_xspi_nor))
+#define FLASH_DEVICE_BASE DT_REG_ADDR_BY_IDX(DT_INST(0, st_stm32_xspi),1) 
```

Comment:
fix in original commit

Reply:
do you mean squashing 0adf7c02d7d3bac24c36986ec4dd8a59b7fb1fd6 and 22d64bceca08cbd5be134a68d0fc36b308b9bcf8 ?